pkg/plugins/platform/router.go

```diff
@@ -84,8 +86,49 @@ func StartDdevRouter() {
 	// run docker-compose up -d in the newly created directory
 	err = dockerutil.ComposeCmd([]string{dest}, "-p", routerProjectName, "up", "-d")
 	if err != nil {
-		log.Fatalf("Could not start router: %v", err)
+		return fmt.Errorf("failed to start ddev-router: %v", err)
+	}
+
+	fmt.Println("Starting service health checks...")
+
+	// ensure we have a happy router
+	label := map[string]string{"com.docker.compose.service": "ddev-router"}
+	err = dockerutil.ContainerWait(35, label)
+	if err != nil {
+		return fmt.Errorf("ddev-router failed to become ready: %v", err)
 	}
+
+	return nil
+}
+
+// PrintRouterStatus outputs router status and warning if not
+// running or healthy, as applicable.
+func PrintRouterStatus() string {
+	var status string
+
+	badRouter := "\nThe router is not currently running. Your sites are likely inaccessible at this time.\nTry running 'ddev start' on a site to recreate the router."
+
+	label := map[string]string{"com.docker.compose.service": "ddev-router"}
+	container, err := dockerutil.FindContainerByLabels(label)
+
+	if err != nil {
+		status = color.RedString(SiteNotFound) + badRouter
+	} else {
+		status = dockerutil.GetContainerHealth(container)
+	}
+
+	switch status {
```

Comment:
I suspect this code block could be simplified by just setting a default color of red, etc.